Bottom: 43088c8e47e6b7accab3ff2ded06ad636de12f26
Top:    8577185a6c5054004977d6b3922e1e231c0406bf
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-07-15 13:47:21 +0800

Refresh of vhost-forbid-iotlb

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 11d2f55..711d6e2 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -958,6 +958,10 @@ int vhost_process_iotlb_msg(struct vhost_dev *dev,
 		vhost_iotlb_notify_vq(dev, msg);
 		break;
 	case VHOST_IOTLB_INVALIDATE:
+		if (!dev->iotlb) {
+			ret = -EFAULT;
+			break;
+		}
 		vhost_del_umem_range(dev->iotlb, msg->iova,
 				     msg->iova + msg->size - 1);
 		break;
